### ISSUE TITLE
Nextcloud integration

### ISSUE THREAD

**Issue by nsrosenqvist** [dularion/streama#553]
**Feature Request**

I recently found this project and I saw an [issue](https://github.com/dularion/streama/issues/75) from 2015 where plans were discussed to separate the UI from the backend. I've been looking for a way to conveniently stream content from my Nextcloud instance but there is no straight forward way of doing this. It would be great if a very simple UI could be implemented as a Nextcloud app. Would that be something that the developers are interested in working towards?

Thanks!

**Comment by dularion** [comment]
Oh interesting! The code for the mobile app is actually just that: just a Frontend speaking to the existing server via api. The code for the app can be found here on github too, called treama-app. It's angular with ionic

**Comment by sunjam** [comment]
Here is a link to [Nextcloud app dev documentation](https://docs.nextcloud.com/server/13/developer_manual/app/index.html).  Cool to see both apps use the Rest API!

**Comment by coamtaker618** [comment]
Yes please!  While a movie app on NextCloud does already exist, it's a far cry from Streama.  That said, given that the server side of Streama seems relatively stable, I don't know if this is actually less work from a Dev perspective?  Would you then have two APIs to manage for the apps/clients?  Would there be problems if NextCloud changes their own APIs?

From an admin perspective, it seems great.  It would allow NextCloud to manage files & do all sorts of fancy file stuff--even potentially allow you to add files to Streama that are from a federated account without needing to store it locally!  It's also one less VM/container I need to host, which is a plus.
